Libplanet.Tools/Mpt.cs

```diff
@@ -0,0 +1,88 @@
+#nullable enable
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using Cocona;
+using Cocona.Help;
+using Libplanet.RocksDBStore;
+using Libplanet.Store.Trie;
+
+namespace Libplanet.Tools
+{
+    public class Mpt
+    {
+        private readonly IImmutableDictionary<string, Func<string, IKeyValueStore>>
+            _kvStoreConstructors =
+                ImmutableSortedDictionary<string, Func<string, IKeyValueStore>>.Empty
+                    .Add("default", kvStorePath => new DefaultKeyValueStore(kvStorePath))
+                    .Add("rocksdb", kvStorePath => new RocksDBKeyValueStore(kvStorePath));
+
+        [Command(Description = "Compare two trees via root hash.")]
+        public void Diff(
+            [Argument(
+                Name = "KEY-VALUE-STORE-PATH",
+                Description = "The path where IKeyValueStore implementation was used at.")]
+            string kvStorePath,
+            [Argument(
+                Name = "STATE-ROOT-HASH",
+                Description = "The state root hash to compare.")]
+            string stateRootHashHex,
+            [Argument(
+                Name = "OTHER-STATE-ROOT-HASH",
+                Description = "Another state root hash to compare.")]
+            string otherStateRootHashHex,
+            [Option(
+                't',
+                Description = "The type of the key-value store stored" +
+                              " at the given KEY-VALUE-STORE-PATH argument. " +
+                              "It should be among in 'default' or 'rocksdb'.")]
```

Comment:
As it's code duplication (the same keys are appeared twice in this file), it might be better to have a separated command to list all available choices, and mention the command here… but I am not sure.

Anyway, I wrote a feature request which address this kind of problem to Cocona: <https://github.com/mayuki/Cocona/issues/27>.